docs/operator-manual/webhook.md

```diff
@@ -12,6 +12,35 @@ a Git webhook for GitHub, but the same process should be applicable to other pro
     the same. A hook event for a push to branch `x` will trigger a refresh for an app pointing at the same repo with
     `targetRevision: refs/tags/x`.
 
+### Webhook Ingress
+
+The applicationset-controller is responsible for answering to webhook calls so a separate ingress has to be setup for it.
```

Comment:
both the server and applicationset controller are responsible have a webhook handler.